generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -108,7 +108,7 @@ module.exports = (options) => ({
         <%_ } _%>
         new HtmlWebpackPlugin({
             template: './<%= MAIN_SRC_DIR %>index.html',
-            chunks: ['vendors', 'polyfills', 'main', 'global'],
+            chunks: ['vendors', 'polyfills', 'vendor', 'main'],
```

Comment:
do we really require this new `vendor` chunk as third party packages are also contained inside `vendors` chunk? If yes, then, it would be good to provide better name to clarify content that it can contain.

Reply:
Yes we need the new `vendor` chunk as the vendor CSS is not extracted into the `vendors` chunk. As the `vendor` chunk contains only the import of fontawesome icons and the vendor CSS we could name it `vendor-styles`. What do you think?

Reply:
Can't we merge those two ?